scipy/optimize/_lsq/least_squares.py

```diff
@@ -255,7 +255,10 @@ def least_squares(
         ``fun(x, *args, **kwargs)``, i.e., the minimization proceeds with
         respect to its first argument. The argument ``x`` passed to this
         function is an ndarray of shape (n,) (never a scalar, even for n=1).
-        It must return a 1-d array_like of shape (m,) or a scalar.
+        It must return a 1-d array_like of shape (m,) or a scalar. If the
+        argument ``x`` is complex or the function ``fun`` returns complex
+        residuals, it must be wrapped in a real function of real arguments,
+        as shown in the `Examples`_ section.
```

Comment:
This link doesn't work right in the CircleCI build:

https://1324-1460385-gh.circle-artifacts.com/0/home/ubuntu/scipy/doc/build/html-scipyorg/generated/scipy.optimize.least_squares.html#id22